elements/pfe-progress-indicator/src/pfe-progress-indicator.scss

```diff
@@ -0,0 +1,67 @@
+@import "../../pfe-sass/pfe-sass";
+:host {
+}
+
+:host([hidden]) {
+  display: none;
+}
+
+:host([pfe-indeterminate]) {
+    animation:spin 1s linear infinite;
+    -webkit-animation:spin 1s linear infinite;
+    -moz-animation:spin 1s linear infinite;
+
+    border-bottom: 4px solid rgba(0,0,0,0.25);
+    border-left: 4px solid rgba(0,0,0,0.25);
+    border-radius: 100%;
+    border-right: 4px solid rgba(0,0,0,0.25);
+    border-top: 4px solid rgba(0,0,0,0.75);
+
+    display: inline-block;
+    height: 2rem;
+    margin: 0 auto;
+    position: relative;
+    width: 2rem;
+    vertical-align: middle;
+    visibility: visible;
+}
+
+@-ms-keyframes spin {
```

Comment:
@Djfaucette Don't worry about prefixing, we're adding autoprefixer very soon (there's a branch almost through review with this update).